config/schema/indexes/govuk.json

```diff
@@ -24,6 +24,7 @@
     "service_manual_guide",
     "service_manual_topic",
     "service_standard_report",
+    "residential_property_tribunal_decision",
```

Comment:
This should be in alphabetical order or @brenetic will be sad.

Reply:
👍 😢

Reply:
Had alphabetical blindness there. Fixed now.